kolibri/plugins/style_guide/assets/src/views/content/api_table.vue

```diff
@@ -0,0 +1,101 @@
+<template>
```

Comment:
I don't want to break this up since they will always be used together. Since this doesn't contain only tables (it also includes a name and a description), renamed it to component_api.vue.